app/controllers/concerns/vlp_doc.rb

```diff
@@ -98,7 +98,23 @@ def sensitive_info_changed?(role)
 
   def native_status_changed?(role)
     return unless role
-    params_hash = params.permit("tribal_id").to_h
-    role.person.send("tribal_id") != params_hash["tribal_id"]
+
+    if EnrollRegistry.feature_enabled?(:indian_alaskan_tribe_codes)
+      tribal_state = params.dig("person", "tribal_state")
+      tribe_codes =  params.dig("person", "tribe_codes")
+      tribal_name = params.dig("person", "tribal_name")
+      tribal_id = params.dig("person", "tribal_id")
+
+      if tribal_state == EnrollRegistry[:enroll_app].setting(:state_abbreviation).item
+        role.person.send("tribe_codes") != tribe_codes
```

Comment:
can this be changed to 
```suggestion
        role.person.tribe_codes != tribe_codes
```